Re-written Cumulocity Platform Introduction (#2466)
SoftwareAG/c8y-docs: content/concepts/concepts-introduction-bundle/device-management.md

---
weight: 30

> title: Device Management

Reviewer: ```suggestion
title: Device management
```

---

> Effectively managing devices is a critical yet often overlooked aspect of deploying IoT solutions. Without proper device management in place, it will become increasingly difficult to keep your devices healthy and up-to-date over time, leading to security risks, operational failures, and increased maintenance costs. This is especially true once you start scaling up your fleet of devices as well as adding new functionalites and thereby complexity to your deployment.

Reviewer: ```suggestion
Effectively managing devices is a critical yet often overlooked aspect of deploying IoT solutions. Without proper device management in place, it will become increasingly difficult to keep your devices healthy and up-to-date over time, leading to security risks, operational failures, and increased maintenance costs. This is especially true once you start scaling up your fleet of devices as well as adding new functionalities and thereby complexity to your deployment.
```

> Connecting equipment to the internet requires secure and robust device management practices. Cumulocity IoT greatly reduces the complexity and time required to manage a heterogeneous fleet of connected devices across its lifecycle through its comprehensive device management features:

Reviewer: ```suggestion
Connecting equipment to the internet requires secure and robust device management practices. {{< product-c8y-iot >}} greatly reduces the complexity and time required to manage a heterogeneous fleet of connected devices across its lifecycle through its comprehensive device management features:
```

- **[Device onboarding](/device-management-application/registering-devices/)**: Register and integrate any number of devices in just one go.
- **Over-the-air (OTA) [firmware](/device-management-application/managing-device-data/#managing-firmware) and [software](/device-management-application/managing-device-data/#managing-software) updates**: Keep your devices secure and up to date by efficiently rolling out the latest available software or firmware version.
- **[Configuration management](/device-management-application/managing-device-data/#managing-configurations)**: Easily change the settings of your devices by applying new configurations.
- **[Connection](/device-management-application/monitoring-and-controlling-devices/#connection-monitoring) and [connectivity](/device-management-application/connectivity/) monitoring**: Quickly identify devices that stopped communicating and identify the underlying issue.
- **[Remote troubleshooting](/cloud-remote-access/cra-general-aspects/)**: Use a set of tools to resolve identified issues and minimize downtimes.

> - **[Device Replacement](/device-management-application/viewing-all-devices/#to-replace-a-device)**: Replace your physical devices without loosing the data history.

Reviewer: ```suggestion
- **[Device replacement](/device-management-application/viewing-all-devices/#to-replace-a-device)**: Replace your physical devices without loosing the data history.
```
